src/integTest/kotlin/io/snyk/SnykBulkFileListenerTest.kt

```diff
@@ -0,0 +1,56 @@
+package io.snyk
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.components.service
+import com.intellij.openapi.fileEditor.FileDocumentManager
+import com.intellij.openapi.roots.ProjectRootManager
+import com.intellij.psi.PsiDocumentManager
+import com.intellij.testFramework.PsiTestUtil
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+import io.snyk.plugin.ui.toolwindow.SnykToolWindowPanel
+import org.junit.Test
+import snyk.oss.OssResult
+
+class SnykBulkFileListenerTest : BasePlatformTestCase() {
+
+    @Test
+    fun testBuildFileChanged() {
```

Comment:
neat pick: I recommend to use following schema for test methods: `test<methodUnderTest>_<testedStated>_<expectedBehaviour>`.
Since IntelliJ platform uses JUnit3 as test framework `test*` prefix for method is necessary.

For this case it could be something like that: `testCurrentOssResults_shouldDropCachedResult_whenBuildFileChanged`

Reply:
Like the schema, despite Idea's inspection is complaining about naming, it make tests more "speaking".